apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/loading.tsx

```diff
@@ -1,7 +1,12 @@
-import Loader from '@/components/Loader/Loader'
+import React from 'react'
+import { Loader } from '@gdh/ui-system'
+import { I18nNamespaces } from '@/i18n'
+import { useTranslations } from 'next-intl'
 
 const Loading = () => {
-  return <Loader />
+  const t = useTranslations(I18nNamespaces.Index)
```

Comment:
**suggestion**: Consider adding a default translation key to handle cases where the translation might be missing.